x-pack/plugin/sql/jdbc/src/test/java/org/elasticsearch/xpack/sql/jdbc/JdbcResultSetMetaDataTests.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+package org.elasticsearch.xpack.sql.jdbc;
+
+import org.elasticsearch.test.ESTestCase;
+
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.elasticsearch.xpack.sql.client.StringUtils.EMPTY;
+
+public class JdbcResultSetMetaDataTests extends ESTestCase {
+    
+    private final List<JdbcColumnInfo> columns = Arrays.asList(
+                new JdbcColumnInfo("test_keyword", EsType.KEYWORD, EMPTY, EMPTY, EMPTY, EMPTY, 0),
+                new JdbcColumnInfo("test_integer", EsType.INTEGER, EMPTY, EMPTY, EMPTY, EMPTY, 11),
+                new JdbcColumnInfo("test_double", EsType.DOUBLE, EMPTY, EMPTY, EMPTY, EMPTY, 25)
+            );
```

Comment:
Please add an example like `new JdbcColumnInfo("test_long", EsType.DOUBLE, EMPTY, EMPTY, EMPTY, "long", 25)` to make clear that the label overrides the name.

Reply:
You're right. Extended the test. Thanks.